streams/src/main/java/org/apache/kafka/streams/kstream/internals/foreignkeyjoin/SubscriptionResponseWrapperSerde.java

```diff
@@ -21,26 +21,14 @@
 import org.apache.kafka.common.serialization.Serializer;
 
 import java.nio.ByteBuffer;
-import java.util.Map;
 
 public class SubscriptionResponseWrapperSerde<V> implements Serde<SubscriptionResponseWrapper<V>> {
     private final SubscriptionResponseWrapperSerializer<V> serializer;
     private final SubscriptionResponseWrapperDeserializer<V> deserializer;
-    public static int versionBits = 7;
 
     public SubscriptionResponseWrapperSerde(final Serde<V> foreignValueSerde) {
-        this.serializer = new SubscriptionResponseWrapperSerializer<>(foreignValueSerde.serializer());
-        this.deserializer = new SubscriptionResponseWrapperDeserializer<>(foreignValueSerde.deserializer());
-    }
-
-    @Override
-    public void configure(final Map configs, final boolean isKey) {
-
-    }
-
-    @Override
-    public void close() {
-
+        serializer = new SubscriptionResponseWrapperSerializer<>(foreignValueSerde.serializer());
+        deserializer = new SubscriptionResponseWrapperDeserializer<>(foreignValueSerde.deserializer());
```

Comment:
This class needs some way to deal with the fact that `foreignValueSerde` is nullable (and should use the default key serde when it is null).